.eslintrc.json

```diff
@@ -52,7 +68,12 @@
       "files": ["*.ts", "*.tsx", "!*.spec.ts", "!*.test*.ts"],
       "extends": ["plugin:@nx/typescript"],
 
-      "rules": {}
+      "rules": {
+        "@typescript-eslint/no-unused-vars": [
+          "error",
+          { "ignoreRestSiblings": true }
```

Comment:
I added this so that we can do something like this without eslint upset

```typescript
// shouldn't get the error that we have an unused variable
const { ignoreVar, ...rest } = obj
return obj;
```

Reply:
https://github.com/ForgeRock/forgerock-javascript-sdk/pull/357/files#diff-120136b5b42331172e5db5ad243029d9e6de97877674855266b3382dd2fc68f9R123 

for example.

Reply:
my code example was wrong whichi is why its confusing, ill fix it

Reply:
Oh, so you are wanting everything but a property, so you extract the properties you don't want, and then collect everything else into a `...rest`, yeah? If so, that makes sense.